src/terra-dev-site/test/application-status-overlay/StatusOverlayAllProps.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ApplicationStatusOverlay from '../../../../lib/application-status-overlay/ApplicationStatusOverlay';
+import ApplicationStatusOverlayProvider from '../../../../lib/application-status-overlay/ApplicationStatusOverlayProvider';
+
+const ApplicationStatusOverlayAllProps = () => {
+  const StatusViewButtons = [
+    {
+      text: 'Action 1',
+      key: 1,
+      size: 'medium',
```

Comment:
`size` is no longer a valid button prop

Reply:
Removed here : https://github.com/cerner/terra-application/pull/47/commits/62b43a41763ab55e834f07ce82e374ac33e0b409